tests/PHPStan/Analyser/data/non-empty-string-substr.php

```diff
@@ -54,7 +54,7 @@ public function doMbSubstr(string $s, $nonEmpty, $positiveInt, $postiveRange, $n
 		assertType('string', mb_substr($s, 0, $positiveInt));
 		assertType('non-empty-string', mb_substr($nonEmpty, 0, $positiveInt));
 
-		assertType('non-falsy-string', mb_substr("déjà_vu", 0, $positiveInt));
+		assertType('non-empty-string', mb_substr("déjà_vu", 0, $positiveInt));
```

Comment:
We could keep `non-falsy-string` here since there is no `0` in the string constant @staabm

Reply:
I also though about that. I think such a case is pretty unlikely and supporting it would mean a lot of complication of the extension.

therefore I decided against supporting that